internal/v4/api.go

```diff
@@ -308,18 +308,40 @@ func (h *ReqHandler) puttableBaseEntityHandler(get baseEntityHandlerFunc, handle
 	})
 }
 
-func (h *ReqHandler) updateBaseEntity(id *router.ResolvedURL, fields map[string]interface{}) error {
+var testAddAuditCallback func(e audit.Entry)
+
+func (h *ReqHandler) processEntries(entries []interface{}) {
```

Comment:
Are these realistically ever going to be anything other than audit log entries? probably better to use the type system here and not do unnecessary type assertions.